### DISCUSSION TITLE
Is there any readme file for multi head interface branch?

### DISCUSSION THREAD

**Discussion by turbosonics** [ACEsuit/mace#543] [Q&A] [answered]
I compiled multi-head-interface branch using 
`git clone --branch=multi-head-interface --depth=1 https://github.com/ACEsuit/mace.git`

I hope to use multi-head-interface branch to conduct series of trainings that are multiple steps of fine tuning, and I just hope to learn how to use multi-head-interface branch, but I couldn't find any materials... 

Plus, what is difference between multi-head-interface branch and multi-head-finetune branch?

**Comment by ilyes319** [comment, marked as answer]
There is no documentation yet as it is a very beta feature. To use it, you can run the same finetuning script as in the main branch https://mace-docs.readthedocs.io/en/latest/guide/finetuning.html. Please use the following model as foundation model: https://github.com/ACEsuit/mace-mp/releases/download/mace_mp_0b/mace_agnesi_medium.model.
